docker/entrypoint.sh

```
@@ -3,7 +3,7 @@
 if [ -f /var/www/html/.env ]; then
   echo "external vars exist."
   # load env vars from .env
-  export $(grep -v '^#' .env | xargs)
+  export $(awk -F[=] 'NF {gsub("\042", "\000");if ($0 ~ /^(#|$)/ ) {next;}; print $1"="$2}' .env)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)